src/Common/tests/System/MockType.cs

```diff
@@ -74,7 +74,6 @@ internal class MockType : Type
         public override MemberInfo[] GetMember(string name, BindingFlags bindingAttr) => throw Unexpected;
         public override MemberInfo[] GetMember(string name, MemberTypes type, BindingFlags bindingAttr) => throw Unexpected;
         protected override TypeCode GetTypeCodeImpl() => throw Unexpected;
-        public override bool IsAssignableFrom(Type c) => throw Unexpected;
```

Comment:
Are the other tests using the MockType broken if this method stays as is in the MockType?

Reply:
I just verified an no - I could just leave one MockType if we don't care about slight pollution

Reply:
I think no BaseMockType would look better. Type has number of non-virtual methods, so there is some impurity by design. If there is ever a test that actually wants IsAssignableFrom to throw, it can have a specialized Mock type that is just for the given test.

Reply:
Ok